gulpfile.js

```diff
@@ -13,7 +13,7 @@ const depcheck = require('depcheck');
 const fs = require('fs');
 const glob = require('glob');
 const gulp = require('gulp');
-const mocha = require('gulp-mocha');
+const mocha = require('gulp-spawn-mocha');
```

Comment:
Yes, we should always use gulp-spawn-mocha. gulp-mocha's behavior is undefined if you call it more than once.